web/src/stream_topic_history.ts

```diff
@@ -31,6 +33,17 @@ export function stream_has_topics(stream_id: number): boolean {
     return history.has_topics();
 }
 
+export function stream_has_resolved_topics(stream_id: number): boolean {
```

Comment:
So this probably deserves a comment at least -- it's only checking the locally available history. So either we expect this is only to be called if we know full history for the channel has been fetched, in which case we should have an assert, or we want a comment or name like `stream_has_locally_available_resolved_topics`.